### PR TITLE
Fix #805 : Highfi: ProfileEditActivity

### LINKED ISSUE #805: Highfi: ProfileEditActivity
<FILE NAME> needs to be pixel perfect with the mocks.
Portrait Mock: https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/1c3f250f-32e5-4299-8187-75d7c3dbe6b6/Settings-1-Profile-Chantel-Disabled-5/
Landscape Mock: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/5b8a8202-6081-4855-8c49-94a8a5b4c559/RG-MN-Register-Empty-35/

Mocks have been created for HDPI Devices.
Make sure that the screen passes the accessibility test also.
For reference you can checkout: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by Scarlet0103**
## Explanation
Fixes #805
Added shadow effect and made evey element to match the mocks

## Mocks
Portrait Mock: https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/1c3f250f-32e5-4299-8187-75d7c3dbe6b6/Settings-1-Profile-Chantel-Disabled-5/
Landscape Mock: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/5b8a8202-6081-4855-8c49-94a8a5b4c559/RG-MN-Register-Empty-35/

## Screenshots

Screenshot_2020-03-18-16-16-40-592_org oppia app
Screenshot_2020-03-18-16-16-49-405_org oppia app
Screenshot_2020-03-18-16-16-54-514_org oppia app

 
## Checklist
- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by nikitamarysolomanpvt** [comment]
@Scarlet0103 Please add mock links in PR description.

**Comment by Scarlet0103** [comment]
@nikitamarysolomanpvt updated the comment thanks...

**Comment by nikitamarysolomanpvt** [comment]
> @nikitamarysolomanpvt updated the comment thanks...

Ack

File: app/src/main/res/layout-land/profile_edit_activity.xml [hunk 1]
```diff
@@ -142,8 +142,8 @@
       <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginTop="24dp"
-        android:layout_marginBottom="8dp"
+        android:layout_marginTop="60dp"
```
Comment (nikitamarysolomanpvt) [inline app/src/main/res/layout-land/profile_edit_activity.xml:150]: Please add text color to profile_edit_allow_download_sub.

File: app/src/main/res/layout/profile_edit_activity.xml [hunk 2]
```diff
@@ -223,6 +221,8 @@
         android:layout_marginStart="28dp"
         android:layout_marginTop="28dp"
         android:layout_marginBottom="36dp"
+        android:elevation="2dp"
```
Comment (nikitamarysolomanpvt) [inline app/src/main/res/layout/profile_edit_activity.xml:231]: Please add text color to profile_edit_allow_download_sub as per mock.

File: app/src/main/res/layout-land/profile_edit_activity.xml (hunk 1 above)
Comment (Scarlet0103) [reply app/src/main/res/layout-land/profile_edit_activity.xml]: The text color code is #00000061 as per the mock. This color very light and isnt showing. Can you help me with this?

File: app/src/main/res/layout-land/profile_edit_activity.xml [hunk 3]
```diff
@@ -119,6 +119,7 @@
                 android:layout_height="wrap_content"
                 android:fontFamily="sans-serif"
                 android:text="@string/profile_edit_allow_download_sub"
+                android:textColor="#61000000"
```
Comment (veena14cs) [inline app/src/main/res/layout-land/profile_edit_activity.xml]: Please add in colors file and use it here.

File: app/src/main/res/layout/profile_edit_activity.xml [hunk 4]
```diff
@@ -113,6 +113,7 @@
               android:layout_height="wrap_content"
               android:fontFamily="sans-serif"
               android:text="@string/profile_edit_allow_download_sub"
+              android:textColor="#61000000"
```
Comment (veena14cs) [inline app/src/main/res/layout/profile_edit_activity.xml]: Ditto.

File: app/src/main/res/layout-land/profile_edit_activity.xml (hunk 3 above)
Comment (Scarlet0103) [reply app/src/main/res/layout-land/profile_edit_activity.xml]: what should I name the color. Can I name it oppia_transparent_text_color?

**Review by veena14cs — COMMENTED** [review]
Change all the buttons height to wrap content and keep min height as 48dp.
Change textsizes to sp's instead of dp.
When you define padding start, also define padding end to support  RTL.

**Comment by Scarlet0103** [comment]
@veena14cs I have updated as you mentioned can you review it?  Thanks...

**Comment by veena14cs** [comment]
Code LGTM. One issue is when I change to landscape User name is change to Oppia in toolbar. Can you please check it.

**Review by veena14cs — APPROVED** [review]
Code LGTM. One issue is when I change to landscape User name is change to Oppia in toolbar. Can you please check it.

**Comment by rt4914** [comment]
> Code LGTM. One issue is when I change to landscape User name is change to Oppia in toolbar. Can you please check it.

I will check this as it is lowfi issue. For now, I will merge this PR.

**Pull request merged**